ray-operator/controllers/raycluster_controller.go

```diff
@@ -296,15 +327,20 @@ func (r *RayClusterReconciler) reconcilePods(instance *rayiov1alpha1.RayCluster)
 					}
 					log.Info("reconcilePods", "workers specified to delete was already deleted ", pod.Name)
 				}
+				//// SriramQ: Shouldn't this be in the else part of the above if?
```

Comment:
The difference is whether we want to emit event when NOT_FOUND issue happens. User may type invalid names, in such cases, we may not want to include not_found pods. I feel move this line to else logic makes more sense?